sensor_msgs/Image supervoxel 
---
float32[] feature
